typings/index.d.ts

```diff
@@ -807,6 +807,24 @@ declare module 'discord.js' {
     public valueOf(): string;
   }
 
+  export class GuildPreview extends Base {
+    constructor(client: Client, data: object);
+    public approximateMemberCount: number;
+    public approximatePresenceCount: number;
+    public description?: string;
+    public emojis: GuildEmojiManager;
+    public features: GuildFeatures;
+    public id: string;
+    public name: string;
+
+    public discoverySplashURL(options?: ImageURLOptions): string | null;
+    public iconURL(options?: ImageURLOptions & { dynamic?: boolean }): string | null;
+    public splashURL(options?: ImageURLOptions): string | null;
+    public fetch(): Promise<GuildPreview>;
+    public toJSON(): object;
+    public toString(): string;
+    private _patch(data: object);
```

Comment:
We don't really document private methods that are *purely* used internally.

And should we, they are usually above public ones.

Reply:
Ah, sorry. I wasn't aware, thank you for letting me know. I've removed it from the typings.